web/app.js

```diff
@@ -1906,10 +1907,15 @@ const PDFViewerApplication = {
   },
 
   bindWindowEvents() {
+    if (this._windowAbortController) {
```

Comment:
This method is sort-of "public", whereas the PresentationMode-methods are *private* and cannot be accidentally accessed.
However, your question made me look at this again and I think it probably cannot hurt to avoid a future (implementation) bug that'd cause multiple listeners to be accidentally added; thank you!